contracts/deployer/Create3.sol

```diff
@@ -2,35 +2,34 @@
 pragma solidity ^0.8.20;
 
 /**
-  @title A library for deploying contracts EIP-3171 style.
-  @author Agustin Aguilar <aa@horizon.io>
-*/
+ * @title A library for deploying contracts EIP-3171 style.
+ *   @author Agustin Aguilar <aa@horizon.io>
+ */
 library Create3 {
     /**
-    @notice The bytecode for a contract that proxies the creation of another contract
-    @dev If this code is deployed using CREATE2 it can be used to decouple `creationCode` from the child contract address
+     * @notice The bytecode for a contract that proxies the creation of another contract
+     * @dev If this code is deployed using CREATE2 it can be used to decouple `creationCode` from the child contract address
+     * 
+     *   0x67363d3d37363d34f03d5260086018f3:
+     *   0x00  0x67  0x67XXXXXXXXXXXXXXXX  PUSH8 bytecode  0x363d3d37363d34f0
+     *   0x01  0x3d  0x3d                  RETURNDATASIZE  0 0x363d3d37363d34f0
+     *   0x02  0x52  0x52                  MSTORE
+     *   0x03  0x60  0x6008                PUSH1 08        8
+     *   0x04  0x60  0x6018                PUSH1 18        24 8
+     *   0x05  0xf3  0xf3                  RETURN
+     * 
+     *   0x363d3d37363d34f0:
+     *   0x00  0x36  0x36                  CALLDATASIZE    cds
+     *   0x01  0x3d  0x3d                  RETURNDATASIZE  0 cds
+     *   0x02  0x3d  0x3d                  RETURNDATASIZE  0 0 cds
+     *   0x03  0x37  0x37                  CALLDATACOPY
+     *   0x04  0x36  0x36                  CALLDATASIZE    cds
+     *   0x05  0x3d  0x3d                  RETURNDATASIZE  0 cds
+     *   0x06  0x34  0x34                  CALLVALUE       val 0 cds
+     *   0x07  0xf0  0xf0                  CREATE          addr
+     */
 
-  0x67363d3d37363d34f03d5260086018f3:
-      0x00  0x67  0x67XXXXXXXXXXXXXXXX  PUSH8 bytecode  0x363d3d37363d34f0
-      0x01  0x3d  0x3d                  RETURNDATASIZE  0 0x363d3d37363d34f0
-      0x02  0x52  0x52                  MSTORE
-      0x03  0x60  0x6008                PUSH1 08        8
-      0x04  0x60  0x6018                PUSH1 18        24 8
-      0x05  0xf3  0xf3                  RETURN
-
-  0x363d3d37363d34f0:
-      0x00  0x36  0x36                  CALLDATASIZE    cds
-      0x01  0x3d  0x3d                  RETURNDATASIZE  0 cds
-      0x02  0x3d  0x3d                  RETURNDATASIZE  0 0 cds
-      0x03  0x37  0x37                  CALLDATACOPY
-      0x04  0x36  0x36                  CALLDATASIZE    cds
-      0x05  0x3d  0x3d                  RETURNDATASIZE  0 cds
-      0x06  0x34  0x34                  CALLVALUE       val 0 cds
-      0x07  0xf0  0xf0                  CREATE          addr
-  */
-
-    bytes internal constant PROXY_CHILD_BYTECODE =
-        hex"67_36_3d_3d_37_36_3d_34_f0_3d_52_60_08_60_18_f3";
+    bytes internal constant PROXY_CHILD_BYTECODE = hex"67363d3d37363d34f03d5260086018f3";
```

Comment:
Is this intentional or a part of formatting? Coz the value is also changed.

Reply:
both format works in solidity. value is same